spec/models/member_spec.rb

```diff
@@ -89,10 +89,55 @@
           create(:member, email: nil)
         end.to_not raise_error
       end
+
+      it 'is invalid with the same email with different casing' do
+        member = build(:member, email: 'Foo@eXample.COM')
+        expect(member).to be_invalid
+      end
+
+      it 'cannot save a member with the same email with different casing' do
+        member = build(:member, email: 'Foo@eXample.COM')
+        expect { member.save }.not_to change { Member.count }
+      end
```

Comment:
This previous two specs are redundant and testing Rails code. If a model is invalid it wont be saved when calling `save`.